src/components/navigation/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useMemo } from 'react'
+import { View } from 'react-native'
+import { Box } from '../../structure'
+import { useStyles } from '../../../theme'
+import { Tab } from './Tab/Tab'
+
+export interface TabDescriptor {
+  /**
+   * ID of the tab.
+   */
+  id: string
+  /**
+   * Text of the tab.
+   */
+  label: string
+}
+
+export interface TabsProps {
+  /**
+   * List of tabs.
+   */
+  tabs: [TabDescriptor, ...TabDescriptor[]]
```

Comment:
Why not `TabDescriptor[]`?

Reply:
Both could be valid options... `[TabDescriptor, ...TabDescriptor[]]` means `TabDescriptor[]`, but containing at least one element. Passing an empty array would not be valid.

Reply:
Replaced with `TabDescriptor[]` for readability and usage simplicity.